portal-api/src/v1/routes.js

```diff
@@ -28,25 +28,49 @@ const { ukefDecisionReport, unissuedFacilitiesReport } = require('./controllers/
 const dealImportBssEwcsController = require('./controllers/data-migration/deal-import-bss-ewcs.controller');
 const dealImportGefController = require('./controllers/data-migration/deal-import-gef.controller');
 
-const users = require('./users/routes');
 const { cleanXss, fileUpload } = require('./middleware');
+const checkApiKey = require('./middleware/headers/check-api-key');
 
+const users = require('./users/routes');
 const gef = require('./gef/routes');
 
-const authRouter = express.Router();
 const openRouter = express.Router();
-const authRouterAllowXss = express.Router();
+const authRouter = express.Router();
+
+authRouter.use(cleanXss);
```

Comment:
As mentioned on the call, we should potentially swap this and line 41, just so the checks are done in the same order as previously